cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1367,6 +1367,9 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: ["storage.k8s.io"]
```

Comment:
With the change in RBAC I'd suggest we don't backport it to keep the upgrades simple for patch releases. There isn't a need to backport, right?